<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerSegmentationsSettingsPanel</class>
 <widget class="ctkSettingsPanel" name="qSlicerSegmentationsSettingsPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>292</width>
    <height>113</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SegmentationsSettingsPanel</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="AutoOpacitiesLabel">
     <property name="toolTip">
      <string>Automatically set opacities of the segments when loading from file based on which contains which, so that no segment obscures another</string>
     </property>
     <property name="text">
      <string>Automatic segment opacities:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QCheckBox" name="AutoOpacitiesCheckBox">
     <property name="toolTip">
      <string>Automatically set opacities of the segments when loading from file based on which contains which, so that no segment obscures another</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="SurfaceSmoothingLabel">
     <property name="toolTip">
      <string>Automatically set opacities of the segments when loading from file based on which contains which, so that no segment obscures another</string>
     </property>
     <property name="text">
      <string>Enable surface smoothing by default:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QCheckBox" name="SurfaceSmoothingCheckBox">
     <property name="toolTip">
      <string>Enable surface smoothing during binary labelmap to closed surface conversion in new segmentations. Smoothing improves appearance in 3D views and exported models but makes segment editing considerably slower.</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Default terminology entry:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="ctkPushButton" name="EditDefaultTerminologyEntryPushButton">
     <property name="toolTip">
      <string>This terminology will be used by default for new segments in an empty segmentation.</string>
     </property>
     <property name="text">
      <string>(set)</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Edit hidden segments: </string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="ctkComboBox" name="AllowEditingHiddenSegmentComboBox">
     <property name="toolTip">
      <string>This option controls what the application should do if the user edits a segment that is currently not visible. It is meant to prevent unintentional changes to hidden segments.</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkComboBox</class>
   <extends>QComboBox</extends>
   <header>ctkComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>ctkPushButton</class>
   <extends>QPushButton</extends>
   <header>ctkPushButton.h</header>
  </customwidget>
  <customwidget>
   <class>ctkSettingsPanel</class>
   <extends>QWidget</extends>
   <header>ctkSettingsPanel.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
